nss-day-cohort-41/react-nutshell-brazil-nuts
# Nutshell 2

Zach McWhirter    Nutshell: Brazil-Nuts

Reason for merge: New Feature

Added a feature that allows the user to save online news articles and view them in their content area. Users can also remove  articles they no longer need. The saved articles will be displayed in the order they were saved, descending from oldest to most recent.

To test these features:

Add new article - enter the required information into the form fields (title, synopsis, url). And then click the save article button.

View saved articles - after you create and save your new article, it will appear in the space below the new entry form.

Remove saved article - There will be a large button marked "Remove" on each article card. Simply click the button on the card you wish to remove.

# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have added test instructions that prove my fix is effective or that my feature works
